backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/ConfirmationCodeModel.java

```diff
@@ -0,0 +1,106 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.model;
+import java.time.Instant;
+
+import java.util.Objects;
+
+import org.springframework.core.style.ToStringCreator;
+import org.springframework.hateoas.server.core.Relation;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import ca.gov.dtsstn.cdcp.api.web.model.BaseResourceModel;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.annotation.Nullable;
+@Schema(name = "ConfirmationCode")
+@Relation(collectionRelation = "confirmationCodes", itemRelation = "confirmationCode")
+@JsonPropertyOrder({ "id", "userId","email", "confirmationCode", "codeCreatedDate", "codeExpiryDate", "createdBy", "createdDate", "lastModifiedBy", "lastModifiedDate" })
+public class ConfirmationCodeModel extends BaseResourceModel<ConfirmationCodeModel>{
```

Comment:
need some vertical spacing here

Reply:
should be fixed with the formatter ive used